### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by dsiragusa**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+Thank you for your interest in this project.
+If you wish to submit a bug report, propose a feature or contribute a change,
+please follow the guidelines described in this document.
+
+
+## Bug reports and proposals
+
+We use the [Github issue tracker](https://github.com/synthesio/zconfigcheck/issues) for bugs
```
Comment (ccoVeille) [inline CONTRIBUTING.md:10]:
```suggestion
We use the [GitHub issue tracker](https://github.com/synthesio/zconfigcheck/issues) for bugs
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# zConfigCheck
+
+`zconfigcheck` is a [zconfig](https://github.com/synthesio/zconfig) linter.
```
Comment (ccoVeille) [inline README.md:3]:
```suggestion
`zconfigcheck` is a linter for [zconfig](https://github.com/synthesio/zconfig).
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+It detects a wide range of common mistakes which can lead to unexpected behavior.
+
+This tool can either be used as a `go vet` tool or as a `golangci-lint` plugin.
+See the [dedicated README.md file](golangci/README.md) for more information about using `zconfigcheck` with
+`golangci-lint`.
+
+## Installation
+z
```
Comment (ccoVeille) [inline README.md:11]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+```shell
```
Comment (ccoVeille) [inline README.md:12]:
````suggestion
```console
````

if you use a `$` or display a command line it's `console` not `shell`

`shell` is for the content of `.sh` file

I know, no one knows that

**Review by ccoVeille — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (dsiragusa) [reply CONTRIBUTING.md]: Done

File: README.md (hunk 3 above)
Comment (dsiragusa) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (dsiragusa) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (dsiragusa) [reply README.md]: Done

**Review by ccoVeille — APPROVED** [review]

**Pull request merged**